orangecontrib/network/widgets/OWNxAnalysis.py

```diff
@@ -249,13 +254,12 @@ def set_items(self, items):
         self.mutex.unlock()
 
     def analyze(self):
-        if self.graph is None or not self.btnCommit.isChecked():
+        if self.graph is None:
```

Comment:
This was deleted since `btnCommit` ceased existing with the retouch. I thought Orange's `auto_commit` handles all the cases. I don't know how to test the re-run, since the computation is extremely fast even for _last-fm.net_.

Perhaps @lanzagar can also check, since I think he helped me with this.